clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -155,7 +154,7 @@ def _parse_month_from_visit_code(visit_code: str) -> int:
     Assumes a visit code of the form 'mXXX'.
     """
     try:
-        return int(visit_code[1:])
+        svisit = visit_code[1:]
```

Comment:
In this case, the function never returns anything, right ?

Reply:
I am not sure how this one happened either, as svisit is not referenced elsewhere in the code. I think that at some point, I was experiencing return statements leading to portions of the code not being run, so I removed some return statements in certain locations to attempt to debug and understand what is happening with the code. Yes we should keep the return statement